Fleece/Tree/MutableNode.hh

```diff
@@ -321,6 +325,10 @@ namespace fleece { namespace hashtree {
             return ::operator new(size + capacity*sizeof(NodeRef));
         }
 
+        static void operator delete(void* ptr, unsigned capacity) {
```

Comment:
This is breaking the GCC and Clang builds (see the Travis results.) I think `operator delete` with a `size_t` parameter may be a MSVC-specific thing.